deploy/prod.yaml

```diff
@@ -4,9 +4,9 @@ metadata:
   name: medlemskap-oppslag
   namespace: default
   labels:
-    team: {{team}}
+    team: medlemskap
 spec:
-  image: {{image}}:{{tag}}
+  image: {{ image }}
```

Comment:
Hva betyr denne endringen?

Reply:
Dette ble kopiert inn i repoet før det ble gjort endringer i nais-deploy. Så oppdateringen reflekterer hvordan nye nais-deploy bruker image-tag

Reply:
Hva betyr det, at selve image-taggen endres for hvert bygg?